packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -241,7 +244,11 @@ export class IndexedDbPersistence implements Persistence {
     assert(!this.started, 'IndexedDbPersistence double-started!');
     assert(this.window !== null, "Expected 'window' to be defined");
 
-    return SimpleDb.openOrCreate(this.dbName, SCHEMA_VERSION, createOrUpgradeDb)
+    return SimpleDb.openOrCreate(
+      this.dbName,
+      SCHEMA_VERSION,
+      new SchemaConverter(this.databaseInfo.databaseId)
```

Comment:
You could just pass the `serializer` through here in which case we wouldn't need the `databaseInfo` as a member here anymore.

Reply:
This is done. It allowed me to revert the changes in the IndexedDbPersistence constructor.